packages/core/utils/src/pricing/events.ts

```diff
@@ -0,0 +1,27 @@
+import { buildEventNamesFromEntityName } from "../event-bus"
+import { Modules } from "../modules-sdk"
+
+const eventBaseNames: [
```

Comment:
Do we even need these explicitly typed now? I guess it will be the `PricingEvents` that will be used directly, right?

Reply:
we need it to infer the types of the PricingEvents, so yes